src/integrationTest/resources/integration-test.properties

```diff
@@ -1,5 +1,7 @@
 idam.api.url=http://localhost:${wiremock.server.port:5000}
 idam.s2s-auth.url=http://localhost:${wiremock.server.port:4502}/s2s
 ccd.data-store.host=http://localhost:${wiremock.server.port:4452}
+ccd.data-store.allowed-urls=/ccd/searchCases.*,/ccd/internal/searchCases.*,/ccd/invalid.*
```

Comment:
minor: do we need to prefix with /ccd here? strictly speaking /ccd is not part of DataStore but MCA